test/testenv/secretutil.go

```diff
@@ -123,7 +122,20 @@ func ModifySecretKey(deployment *Deployment, ns string, key string, value string
 		return false
 	}
 	//Modify data
-	data[key] = []byte(EncodeBase64(value))
-	modify := ModifySecretObject(deployment, data, ns)
+	data[key] = []byte(value)
+	modify := ModifySecretObject(deployment, data, ns, secretName)
 	return modify
 }
+
+//GetMountedKey Gets the key mounted on pod
+func GetMountedKey(deployment *Deployment, podName string, key string) string {
```

Comment:
Add a log if value is found and the key of secret we are returning.

Reply:
added